src/com/walmartlabs/lacinia/schema.clj

```diff
@@ -289,6 +290,13 @@
   ([message data]
    (merge {:message message} data)))
 
+(defn url?
+  [url]
+  (try
+    (URL. url)
```

Comment:
URL constructor is deprecated since Java20.

```clojure
(defn valid-url? [s]
  (try
    (let [uri (java.net.URI. s)]
      (.toURL uri)
      true)
    (catch Exception _ false)))
```